### DISCUSSION TITLE
can not upload image use github mobile

### DISCUSSION THREAD

**Discussion by heartacker** [community/community#28075] [Mobile]
![31fc890b3eb7c7ca0f7455fb11aada7](https://user-images.githubusercontent.com/1876302/182407014-64609839-81f7-4697-b02d-5b18db78ad28.jpg)
see the bottom

**Comment by mxie** [comment]
Hi @heartacker - thanks for reaching out to us. I'm sorry to hear you're experiencing issues.

It looks like you're trying to comment or respond to a thread somewhere. Can you provide the steps you took to get here? Thanks!

**Comment by heartacker** [comment]
https://user-images.githubusercontent.com/1876302/182474043-2934ef8a-f66e-408d-b7ff-443a50257c86.mp4
